core/hazelcast-connectors/src/main/java/cache/RemoteNode.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2008-2017, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cache;
+
+import com.hazelcast.cache.ICache;
+import com.hazelcast.config.CacheSimpleConfig;
+import com.hazelcast.config.Config;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+
+import javax.cache.configuration.FactoryBuilder;
+import javax.cache.configuration.MutableCacheEntryListenerConfiguration;
+import javax.cache.event.CacheEntryCreatedListener;
+import javax.cache.event.CacheEntryEvent;
+import javax.cache.event.CacheEntryListenerException;
+import java.io.Serializable;
+
+import static cache.ReadWriteRemoteCache.SINK_CACHE_NAME;
+import static cache.ReadWriteRemoteCache.SOURCE_CACHE_NAME;
+
+public class RemoteNode implements CacheEntryCreatedListener<String, String>, Serializable {
+
+    private static final int ITEM_COUNT = 10;
+
+    private HazelcastInstance instance;
+
+    void start() throws Exception {
+        Config config = new Config();
+        config.addCacheConfig(new CacheSimpleConfig().setName("*Cache"));
+        config.getNetworkConfig().setPort(6701);
+        HazelcastInstance instance = Hazelcast.newHazelcastInstance(config);
+        ICache<Integer, Integer> sourceCache = instance.getCacheManager().getCache(SOURCE_CACHE_NAME);
+        for (int i = 0; i < ITEM_COUNT; i++) {
+            sourceCache.put(i, i);
+        }
+        ICache<String, String> sinkCache = instance.getCacheManager().getCache(SINK_CACHE_NAME);
+        sinkCache.registerCacheEntryListener(new MutableCacheEntryListenerConfiguration<>(
+                FactoryBuilder.factoryOf(new RemoteNode()), null, false, true
+        ));
+    }
+
+    @Override
+    public void onCreated(Iterable<CacheEntryEvent<? extends String, ? extends String>> iterable)
+            throws CacheEntryListenerException {
+        iterable.forEach(e -> System.out.println(e.getKey() + " - " + e.getValue()));
```

Comment:
These print statements don't say where they come from. We should add something like "Entries added to sink:".